pkg/pillar/cmd/zedrouter/networkinstance.go

```diff
@@ -210,6 +210,10 @@ func networkInstanceBridgeDelete(
 	// When bridge and sister interfaces are deleted, code in pbr.go
 	// takes care of deleting the corresponding route tables and ip rules.
 
+	aclArgs := types.AppNetworkACLArgs{IsMgmt: false, BridgeName: status.BridgeName,
```

Comment:
Given the comment above, why do we need this added code?

Reply:
Please update the comment on line 211 so that it matches the code. I think saying something like:
Here we explicitly delete the rules which are tied to the Linux bridge itself and not the rules for specific domU vifs.

Reply:
Sure